### PR TITLE
Add commitlint and commitizen

### PR DESCRIPTION
**Pull request by leite08**
Ref: https://github.com/metriport/metriport-internal/issues/92

### Dependencies

- Upstream: https://github.com/metriport/metriport/pull/62
- Downstream: https://github.com/metriport/metriport/pull/64

### Description

- Add [commitlint](https://github.com/conventional-changelog/commitlint)
- Add [commitizen](https://github.com/commitizen/cz-cli)
- Update README

### Release Plan

- anytime

### CODE REVIEW

**Comment by leite08** [comment]
See https://github.com/metriport/metriport-internal/pull/277

File: README.md [hunk 1]
```diff
@@ -177,6 +180,63 @@ Useful commands:
...
+- fix
+- perf
+- refactor
+- revert
+- style
+- test
+
+Scope is optional, and we can use one of these, or empty (no scope):
+- api
+- dash
```
Comment (Goncharo) [inline README.md:213]: is this supposed to be widget?

**Review by Goncharo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (leite08) [reply README.md:213]: yep, good catch

**Review by Goncharo — APPROVED** [review]
🚢

**Pull request merged**